website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -121,13 +116,42 @@ Use `external-link` when using the Button for external links. In most cases, con
 
 <Hds::Button @color="secondary" @text="Authenticate with GitHub" @icon="external-link" @iconPosition="trailing" @href="https://hashicorp.com" />
 
-## Button set
+## Links in Buttons
+
+!!! Insight
+
+While we support using links within buttons, please strive for the correct semantic use. Learn more about [how semantic use helps make our products more conformant](/components/button?tab=accessibility#button-vs-link).
+!!!
 
-[ButtonSets](/components/button-set) are patterns when multiple Buttons need to be displayed in a single row.
+The Button component accepts an `@href` or `@route` argument, which results in a link with the visual appearance of a button. While we advise against this approach because it can cause confusion for keyboard-only users, there are a few instances where this may be appropriate for improved visual hierarchy, such as:
```

Comment:
```suggestion
The Button component accepts an `@href` or `@route` argument, which results in a link with the visual appearance of a button. While in general we advise against this approach because it can cause confusion for keyboard-only users, there are a few instances where this may be appropriate for improved visual hierarchy, such as:
```

Reply:
Another consideration: above we say "While we support using links as buttons" and here we say "While we advise against this approach". Could it be seen as contradictory?